products/apache-maven.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Apache maven
+layout: product
+category: build
+versionCommand: mvn --version
+permalink: /maven
+alternate_urls:
+-   /apache-maven
+-   /mvn
+releasePolicyLink: https://maven.apache.org/docs/history.html
+changelogTemplate: "https://github.com/apache/maven/releases/tag/v__LATEST__"
+auto:
+-   git: https://github.com/apache/maven.git
+# See https://rubular.com/r/stJ20etRIblK0J for reference
+    regex: 'https://rubular.com/r/hQMqEiUNHQOGZL'
+releases:
+-   releaseCycle: "3.8"
+    eol: false
+    support: 2023-11-24
+    latest: "3.8.6"
+    latestReleaseDate: 2022-06-06
+    releaseDate: 2021-04-04
+
+-   releaseCycle: "3.6"
+    eol: false
+    support: 2023-11-124
+    latest: "3.6.3"
+    latestReleaseDate: 2019-11-25
+    releaseDate: 2018-10-24
+
+-   releaseCycle: "3.5"
+    eol: false
+    support: 2023-11-24
+    latest: "3.5.4"
+    latestReleaseDate: 2018-06-21
+    releaseDate: 2017-04-07
+
+-   releaseCycle: "3.3"
+    eol: false
+    support: 2022-05-19
+    latest: "3.3.9"
+    latestReleaseDate: 2015-11-14
+    releaseDate: 2015-03-18
+
+-   releaseCycle: "3.2"
+    eol: false
+    support: 2021-11-18
+    latest: "3.2.5"
+    latestReleaseDate: 2014-12-20
+    releaseDate: 2014-02-21
+
+-   releaseCycle: "3.0"
+    eol: 2013-02-23
+    support: 2013-02-23
+    latest: "3.0.5"
+    latestReleaseDate: 2013-02-23
+    releaseDate: 2010-10-08
+
+-   releaseCycle: "2.x"
+    eol: 2014-02-18
+    support: 2014-02-18
+    latest: "2.2.1"
+    latestReleaseDate: 2009-11-08
+    releaseDate: 2005-10-20
+
+-   releaseCycle: "1.x"
+    eol: 2014-02-18
+    support: 2014-02-18
+    latest: "1.1"
+    latestReleaseDate: 2007-06-25
+    releaseDate: 2004-07-13
+
+activeSupportColumn: true
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Security Support
+iconSlug: maven
+
+---
+
+> [Apache Maven](https://maven.apache.org/) is a software project management and comprehension tool.
+> Based on the concept of a project object model (POM), Maven can manage a project's build,
+> reporting and documentation from a central piece of information.
+
+>  Maven is a part of the Apache Software Foundation. 
+
+
+See ["What is Maven ?" page](https://maven.apache.org/what-is-maven.html) for more about Maven.
+See ["Features"](https://maven.apache.org/maven-features.html)
+["Maven Releases History"](https://maven.apache.org/docs/history.html)
+["Run Apache Maven"](https://maven.apache.org/run.html)
```

Comment:
```suggestion
```

The 3rd is the only relevant link, and the rest are not needed. The third is already covered by `releasePolicyLink` so we don't need it.

The page is meant for _existing Maven users_, we don't need to talk much about Maven itself, beyond the first paragraph.